ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects_permissions_mgmt.robot

```diff
@@ -76,13 +82,33 @@ Verify User Can Assign Access Permissions To User Groups
     Sleep   5s
     ${USER_C} Should Not Have Access To ${PRJ_USER_B_TITLE}
 
+Verify Project Sharing Does Not Override Dashboard Permissions
+    [Tags]                  Tier1                   Sanity                  ODS-2223
+    # [Setup]                 Run Keywords            Restore Permissions Of The Project
+    # ...                     AND                     Set RHODS Users Group To rhods-users
+    [Setup]                 Set RHODS Users Group To rhods-users
+    Launch Data Science Project Main Page    username=${OCP_ADMIN_USER.USERNAME}    password=${OCP_ADMIN_USER.PASSWORD}
+    ...    ocp_user_auth_type=${OCP_ADMIN_USER.AUTH_TYPE}
+    Assign Admin Permissions To User ${USER_B} in Project ${PRJ_USER_B_TITLE}
+    Assign Edit Permissions To User ${USER_C} in Project ${PRJ_USER_C_TITLE}
+    Remove User From Group    username=${USER_B}    group_name=rhods-users
+    Remove User From Group    username=${USER_B}    group_name=rhods-admins
+    Remove User From Group    username=${USER_C}    group_name=rhods-users
+    Remove User From Group    username=${USER_C}    group_name=rhods-admins
+    Verify No Permissions For User ${USER_B}
+    Verify No Permissions For User ${USER_C}
+    [Teardown]              Run Keywords            Set RHODS Admins Group To system:authenticated
```

Comment:
system:authenticated is not the default admin group. Please use `Set Default Access Groups Settings` in the teardown